client/src/components/annotators/VideoAnnotator.vue

```diff
@@ -78,6 +88,7 @@ export default defineComponent({
       video.currentTime = (frame / props.frameRate) + OnePTSTick;
       data.frame = Math.round(video.currentTime * props.frameRate);
       commonMedia.emitFrame();
+      data.time = `${new Date(video.currentTime * 1000).toISOString().substr(11, 8)} / ${new Date(video.duration * 1000).toISOString().substr(11, 8)}`;
```

Comment:
I think it would be better to expose time as float seconds (maybe even change the name) and let the rendering consumer decide how to format it.

Reply:
Yeah that should work better.